exporters/jaeger/src/test/java/io/opentelemetry/exporters/jaeger/JaegerIntegrationTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.exporters.jaeger;
+
+import static io.restassured.RestAssured.given;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import org.awaitility.Awaitility;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+
+@RunWith(JUnit4.class)
+public class JaegerIntegrationTest {
+
+  private static final int QUERY_PORT = 16686;
+  private static final int COLLECTOR_PORT = 14250;
+  private static final String JAEGER_VERSION = "1.17";
+  private static final String SERVICE_NAME = "E2E-test";
+  private static final String JAEGER_IP = "http://localhost";
+  private final Tracer tracer =
+      OpenTelemetry.getTracerProvider()
+          .get("io.opentelemetry.exporters.jaeger.JaegerIntegrationTest");
```

Comment:
Maybe use `getClass().getCanonicalName()`?

Reply:
Thanks for review @thisthat actually i would like it, but in whole code in project i saw we using string approach